### PR TITLE
[clickhouse] update clickhouse zk cluster replica to 3

### LINKED ISSUE #13954: [bitnami/clickhouse] zookeeper cluster replica is 2
### Name and Version

bitnami/clickhouse 2.0.0

### What is the problem this feature will solve?

zookeeper cluster replica is 2 will cause cluster split brain, zookeeper cluster least replica is 3.

### What is the feature you are proposing to solve the problem?

zookeeper cluster error

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by lusson-luo**
Signed-off-by: lusson <lusson@foxmail.com>

### Description of the change

zookeeper cluster replica is 2 will cause cluster split brain, zookeeper cluster least replica is 3.

### Benefits

zookeeper cluster No split-brain

### Possible drawbacks

no

### Applicable issues

  - fixes #13954

### Additional information

no

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/clickhouse/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ name: clickhouse
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/clickhouse
   - https://github.com/ClickHouse/ClickHouse
-version: 2.1.1
+version: 2.1.2
```
Comment (migruiz4) [inline bitnami/clickhouse/Chart.yaml:28]: Could you please bump a minor version? (2.2.0)

I don't think a major would be needed as no backward compatibility is broken, but modifying the number of replicas shouldn't be included in a patch version.

Users may be surprised if upgrading from 2.1.1 to 2.1.2 changed the number of pods in their deployment.

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Thank you for your contribution @lusson-luo! Could you please take a look at my comment?

File: bitnami/clickhouse/Chart.yaml (hunk 1 above)
Comment (lusson-luo) [reply bitnami/clickhouse/Chart.yaml:28]: ok, i will tag 2.2.0

**Review by migruiz4 — APPROVED** [review]
Thank you for applying the suggested change, looks good to me!

**Pull request merged**